__tests__/Ship.test.js

```diff
@@ -0,0 +1,84 @@
+/*global describe it expect */
+const Ship = require('../src/Ship.js');
+const Port = require('../src/Port.js');
+const Itinerary = require('../src/Itinerary.js');
+
+describe('Ship', () => {
```

Comment:
You have odd indentation throughout this file again, sometimes 3, 5 or something else. Rather than write everything all over again for you here, please refer to the rules above on how this should look, or even better install prettier and have that do the job for you :)